### PR TITLE
Reduce code duplication in the TypedArray implementations

### PR DESCRIPTION
**Pull request by mmatyas**

### CODE REVIEW

**Comment by mmatyas** [comment]
Ok, merged. I see no changes in the binary size.

**Comment by zherczeg** [comment]
So the gain is maintenance. Ok for me regardless.

**Comment by mmatyas** [comment]
Ok, updated.

File: jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-float32array-prototype.inc.h [hunk 1]
```diff
@@ -17,20 +17,10 @@
...
-/* ES2015 22.2.3.4 */
-OBJECT_VALUE (LIT_MAGIC_STRING_CONSTRUCTOR,
-              ECMA_BUILTIN_ID_FLOAT32ARRAY,
-              ECMA_PROPERTY_CONFIGURABLE_WRITABLE)
-
-/* ES2015 22.2.6.1 */
-NUMBER_VALUE (LIT_MAGIC_STRING_BYTES_PER_ELEMENT_U,
-              4,
-              ECMA_PROPERTY_FIXED)
+#define TYPEDARRAY_BYTES_PER_ELEMENT 4
```
[the remark this answers is not in the document]
Comment (mmatyas) [reply jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-float32array-prototype.inc.h]: Thanks, updated.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**